tests/utils.go

```diff
@@ -219,10 +222,16 @@ func StopVirtualMachineWithTimeout(vm *v1.VirtualMachine, timeout time.Duration)
 	return updatedVM
 }
 
+// StopVirtualMachine
+//
+// Deprecated: Use client.VirtualMachine(vm.Namespace).Stop and libwait instead
 func StopVirtualMachine(vm *v1.VirtualMachine) *v1.VirtualMachine {
 	return StopVirtualMachineWithTimeout(vm, time.Second*300)
 }
 
+// StartVirtualMachine
+//
+// Deprecated: Use client.VirtualMachine(vm.Namespace).start and libwait instead
```

Comment:
Depending on the count of the usages for this function this might not be redundant but rather convenient and safer. Do we know how many usages of this will remain after we apply the optimization of using the right strategy?

 BTW, We should update the function as you describe (use the libwait instead of the `Eventually`).

Reply:
I don't see a reason to keep this function if we have an API call that does the same. As I see it, this is code duplication.

Reply:
When I wrote this PR, these functions were implemented in a completely different way. If we want to keep it so that it also waits for the VM to be ready, we should at least change its name to reflect that.

Reply:
I agree that we need to give here more clarity. Maybe godoc will be sufficient